packages/name-controller/src/utils.test.ts

```diff
@@ -0,0 +1,54 @@
+import { graphQL } from './util';
+
+describe('Utils', () => {
+  describe('graphQL', () => {
+    const URL_MOCK = 'http://test.com';
+    const QUERY_MOCK = 'query';
+    const VARIABLES_MOCK = { test: 'value' };
+    const DATA_MOCK = { test2: 'value2' };
+    const JSON_MOCK = { data: DATA_MOCK };
+    const RESPONSE_MOCK = { ok: true, json: () => JSON_MOCK } as any;
+
+    it('fetches URL with graphQL body', async () => {
+      const mockFetch = jest.spyOn(global, 'fetch');
```

Comment:
Nit: Is it worth it to dependency-inject `fetch` so you don't have to mock it like this?

Reply:
I agree that would be a great solution if it was a more complex dependency, but this is still a one-line mock for a single fairly fundamental function.

I also wouldn't want to change the API of the production code purely for the sake of unit tests as we've no current or likely future use case for alternate fetch implementations.

Reply:
Sounds good!